### PR TITLE
explicitly show how to change directory to run app

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @LarkAscending

**Comment by abarth** [comment]
LGTM

File: getting-started.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ To write Flutter apps, you will need to install the Flutter SDK. You also need t
 Mac development machine to run and test your apps on Android or iOS. See
 [Flutter Setup](../setup) for instructions on how to set up your development environment.
 
-## Create your first Flutter app
+## Creating your first Flutter app
 
-To create a starter project, open a terminal and run the `flutter` command .
+To create a starter project, open a terminal and run the `flutter create <project name>` command .
```
Comment (LarkAscending) [inline getting-started.md]: Delete space between command and .
( a typo in the original)

File: getting-started.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ app that uses [Material Design](https://www.google.com/design/spec/material-desi
 
 In the project directory, the code for your app is in `myapp/lib/main.dart`.
 
-## Run your Flutter app
+## Running your Flutter app
```
Comment (LarkAscending) [inline getting-started.md]: Agree with gerund form for all these headings. Will look for them also.

File: getting-started.md (hunk 1 above)
Comment (sethladd) [inline getting-started.md]: Good catch

File: getting-started.md [hunk 3]
```diff
@@ -95,7 +98,7 @@ To edit your code and hot reload changes:
 1. Make desired changes in your source code (without stopping the debugger).
 1. In the Debug panel at the bottom, click the Hot Reload button.
 
-Not all source code changes can be hot-reloaded. To fully restart the Dart VM of
```
Comment (LarkAscending) [inline getting-started.md]: This version seems correct to me...compound adjective.

**Pull request merged**